.rubocop.yml

```diff
@@ -13,6 +14,7 @@ AllCops:
 # Metrics/LineLength:
 #   Max: 120
 
+
 Style/StringLiterals:
   Enabled: true
   EnforcedStyle: single_quotes
```

Comment:
I'm confused. This wasn't changed as part of this commit (except for the errant whitespace, which I've removed).